schema/parser.cpp

```diff
@@ -648,7 +640,7 @@ Attribute Parser::parseAttribute(ParserContext *context, const QDomElement &elem
             newAttribute.setAttributeUse(Attribute::Required);
         } else if (use == QLatin1String("prohibited")) {
             qWarning("prohibited attributes are not supported"); // TODO skip parsing the attribute
-                                                                 // altogether
+            // altogether
```

Comment:
the previous indentation made more sense (maybe merge the lines?)

Reply:
Indeed, merging is not possible due to the Qt's clang-formatter 100 char line limit.